File: samples/CS/13-ApplicationModes/RibbonItems.cs
using System;

namespace WinForms.Ribbon
{
    partial class RibbonItems
    {
        public void Init()
        {
            ButtonSwitchToAdvanced.Click += new EventHandler<EventArgs>(_buttonSwitchToAdvanced_ExecuteEvent);
            ButtonSwitchToSimple.Click += new EventHandler<EventArgs>(_buttonSwitchToSimple_ExecuteEvent);
        }

        void _buttonSwitchToAdvanced_ExecuteEvent(object sender, EventArgs e)
        {
            Ribbon.SetModes(1);
        }

        void _buttonSwitchToSimple_ExecuteEvent(object sender, EventArgs e)
        {
            Ribbon.SetModes(0);
        }

        public void Load()
        {
        }

    }
}

File: samples/VB/13-ApplicationModes/RibbonItems.vb
Imports System

Namespace WinForms.Ribbon

    Partial Class RibbonItems

        Public Sub Init()
            AddHandler _ButtonSwitchToAdvanced.Click, AddressOf _buttonSwitchToAdvanced_ExecuteEvent
            AddHandler _ButtonSwitchToSimple.Click, AddressOf _buttonSwitchToSimple_ExecuteEvent
        End Sub

        Private Sub _buttonSwitchToAdvanced_ExecuteEvent(ByVal sender As Object, ByVal e As EventArgs)
            _ribbon.SetModes(1)
        End Sub

        Private Sub _buttonSwitchToSimple_ExecuteEvent(ByVal sender As Object, ByVal e As EventArgs)
            _ribbon.SetModes(0)
        End Sub

    End Class
End Namespace
